apps/src/gamelab/spritelab/eventCommands.js

```diff
@@ -2,18 +2,24 @@ import * as spriteUtils from './spriteUtils';
 
 export const commands = {
   checkTouching(condition, sprite1, sprite2, callback) {
-    spriteUtils.addEvent(
-      condition + 'touch',
-      {sprite1: sprite1, sprite2: sprite2},
-      callback
-    );
+    if (condition === 'when' || condition === 'while') {
```

Comment:
NIT: Does it make sense to move this condition to a function i.e. shouldAddEvent(condition)?